src/components/ActionDataSamplesCard.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  CloseButton,
+  HStack,
+  keyframes,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import { useCallback, useRef } from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import { flags } from "../flags";
+import { DataSamplesView, GestureData } from "../model";
+import { useStore } from "../store";
+import { tourElClassname } from "../tours";
+import RecordingFingerprint from "./RecordingFingerprint";
+import RecordingGraph from "./RecordingGraph";
+
+const flash = keyframes({
```

Comment:
Moved this in from the CSS so we didn't need a folder as I kept getting confused by the index.ts file for just this key file (there's one other but rarely opened).